### ISSUE TITLE
Continue learning carousel

### ISSUE THREAD

**Issue by pedro16v** [catalpainternational/canoe#938]
## Overview
We're currently only showing 1 Course on the `Continue Learning` call-to-action component. The original design intended to have this section as a carousel containing all ongoing Courses.

## Conditions of Satisfaction
- [ ] List all ongoing courses in the `Continue Learning` section

## Resources

| Currently implemented | Intended |
| - | - |
<img width="375" alt="Screenshot 2021-08-09 at 12 21 10" src="https://user-images.githubusercontent.com/2813677/128699233-914d6d45-a72a-4964-a4f2-30f8f7449e9e.png"> | <img width="322" alt="Screenshot 2021-08-09 at 12 21 48" src="https://user-images.githubusercontent.com/2813677/128699243-8e9823dd-8dbb-47c1-a556-2e3286574872.png">

**Comment by pedro16v** [comment]
There's an [ongoing discussion on InVision](https://projects.invisionapp.com/share/8911I5TDCQ5S#/screens/432778428/comments/144680658) of what the arrow next to continue learning should do.

<img width="299" alt="Screenshot 2021-08-09 at 12 31 29" src="https://user-images.githubusercontent.com/2813677/128699840-2d9c798b-1a67-4235-b270-28741c6dcd37.png">

**Comment by pedro16v** [comment]
@andrewwapling @Juliana-t —should the filtering options have any effect on this component? Should we filter out (and potentially hide) options if those don't match the applied filters?

**Comment by pedro16v** [comment]
> There's an [ongoing discussion on InVision](https://projects.invisionapp.com/share/8911I5TDCQ5S#/screens/432778428/comments/144680658) of what the arrow next to continue learning should do.
> 
> <img alt="Screenshot 2021-08-09 at 12 31 29" width="299" src="https://user-images.githubusercontent.com/2813677/128699840-2d9c798b-1a67-4235-b270-28741c6dcd37.png">

@megganeturner for now, we should ignore this navigation control. We'll add it back once we're able to build the destination page.

**Comment by andrewwapling** [comment]
@pedro16v @megganeturner let me know if you require additional design support on this one currently. Ideally it would be great to have this section header link through to a listing screen no dissimilar to a standard course category card listing screen (i.e. ) that surfaces all courses that a user may have commenced though not yet completed. The general UX thinking here is to help users progress with their learning and complete courses. Please see existing indicative design pattern that we could utilise for this purpose: https://invis.io/52Y02L9YTRF#/456762015_Course_Listing_-_Category_-_No_Sub_Sections

**Comment by Juliana-t** [comment]
> @andrewwapling @Juliana-t —should the filtering options have any effect on this component? Should we filter out (and potentially hide) options if those don't match the applied filters?

@pedro16v I think it should - applying a filter and having that not reflected on the 'Continue learning' component, that has a top location is an odd experience. I think applied filters should affect what we show on the 'Continue learning'. @andrewwapling any comments on that?

**Comment by Juliana-t** [comment]
> @pedro16v @megganeturner let me know if you require additional design support on this one currently. Ideally it would be great to have this section header link through to a listing screen no dissimilar to a standard course category card listing screen (i.e. ) that surfaces all courses that a user may have commenced though not yet completed. The general UX thinking here is to help users progress with their learning and complete courses. Please see existing indicative design pattern that we could utilise for this purpose: https://invis.io/52Y02L9YTRF#/456762015_Course_Listing_-_Category_-_No_Sub_Sections

On having a screen (similar to a standard course category card listing) for the continue learning sounds ideal. It is not prioritized but something to add to the roadmap.

**Comment by pedro16v** [comment]
I can't see this on JID staging. Maybe it's not on the published branch?

<img width="374" alt="Screenshot 2021-08-11 at 14 09 11" src="https://user-images.githubusercontent.com/2813677/129034478-5679dbd2-15cf-48fa-92c1-c955a3a72d78.png">

**Comment by andrewwapling** [comment]
> > @andrewwapling @Juliana-t —should the filtering options have any effect on this component? Should we filter out (and potentially hide) options if those don't match the applied filters?
> 
> @pedro16v I think it should - applying a filter and having that not reflected on the 'Continue learning' component, that has a top location is an odd experience. I think applied filters should affect what we show on the 'Continue learning'. @andrewwapling any comments on that?

Yes @Juliana-t @pedro16v @megganeturner , the intention was for any search term or filter applied to only reflect results that match those terms.

**Comment by megganeturner** [comment]
No worries, I can add the filtering in! :) One tangentially related question I had - for the courses listed in 'all courses' we're floating the in-progress ones to the top (with completed courses at the end) - should we still do this? or just allow them to sit in their regular order. with the continue carousel this ordering feels a little redundant to me, but keen to hear your thoughts! @andrewwapling 

![Screen Shot 2021-08-12 at 3 33 38 pm](https://user-images.githubusercontent.com/12974326/129143357-6fafd39d-b079-4841-b129-82deae01c8b3.png)

**Comment by pedro16v** [comment]
That is a very good question, Meggan. And I fear it will trigger a wider discussion due to amount of listings we have. Don't wait for us 😅

**Comment by pedro16v** [comment]
@megganeturner please [check the approved answer](https://github.com/catalpainternational/canoe/discussions/957#discussioncomment-1189833) for your ordering question above.
